### ISSUE TITLE
No diacritics are available in the WiFi password menu

### ISSUE THREAD

**Issue by Witloofboer** [OpenSoaring/OpenVario#4]
Diacritics such as !, #, $, ... are not available in the WiFi password menu. When using a keyboard, the shift key is ignored.

**Comment by August2111** [comment]
Yes, you are right ;-) This is a general problem in all password edit fields in XCSoar/OpenSoar.
At the moment I can only recommend an extremely poor workaround: editing the passphrase directly in the corresponding 'settings' file of the connman settings - this topic also has a very high priority in the next development steps. However, merging OpenSoar into XCSoar has the highest priority, which would free up a lot of developer potential for the near future ;-)

**Comment by Witloofboer** [comment]
Is there a list of things that need to be done? I might be able to help out here and there

**Comment by DanD222** [comment]
> Is there a list of things that need to be done? I might be able to help out here and there

I think the most important thing would be to implement the Larus driver and the second / live wind arrow on the map, or in an infobox.

**Comment by DanD222** [comment]
> However, merging OpenSoar into XCSoar has the highest priority

I would disagree, as I don’t think OpenSoar is mature enough to be considered productio-ready, see for example https://github.com/OpenSoaring/OpenVario/issues/5

My current “workaround” for this is buying a new Cubieboard and not letting OpenSoar anywhere near it.

**Comment by DanD222** [comment]
> Is there a list of things that need to be done? I might be able to help out here and there

It seems that the Larus driver was implemented in XCSoar here https://github.com/XCSoar/XCSoar/pull/1423

So the most important thing now would be to display the “live wind” information on the map and in the wind infobox, in addition to the current wind information in XCSoar.
